docusaurus/docs/themes/transportation/roads.mdx

```diff
@@ -0,0 +1,606 @@
+import ExampleLanesResolutionSegment01 from '!!raw-loader!@site/docs/_examples/transportation/docusaurus/lanes-resolution-segment-01.yaml';
+import ExampleLanesResolutionSegment02 from '!!raw-loader!@site/docs/_examples/transportation/docusaurus/lanes-resolution-segment-02.yaml';
+import ExampleTurnRestriction1Source from '!!raw-loader!@site/docs/_examples/transportation/docusaurus/turn-restriction-01-source.yaml';
+import ExampleTurnRestriction1Target from '!!raw-loader!@site/docs/_examples/transportation/docusaurus/turn-restriction-01-target.yaml';
+import ExampleTurnRestriction1Exit from '!!raw-loader!@site/docs/_examples/transportation/docusaurus/turn-restriction-01-exit.yaml';
+import ExampleTurnRestriction1Connector1 from '!!raw-loader!@site/docs/_examples/transportation/docusaurus/turn-restriction-01-connector1.yaml';
+import ExampleTurnRestriction1Connector2 from '!!raw-loader!@site/docs/_examples/transportation/docusaurus/turn-restriction-01-connector2.yaml';
+import ExampleTurnRestriction1Connector3 from '!!raw-loader!@site/docs/_examples/transportation/docusaurus/turn-restriction-01-connector2.yaml';
+import ExampleTurnRestriction2Source from '!!raw-loader!@site/docs/_examples/transportation/docusaurus/turn-restriction-02-source.yaml';
+import ExampleTurnRestriction2Target from '!!raw-loader!@site/docs/_examples/transportation/docusaurus/turn-restriction-02-target.yaml';
+import ExampleTurnRestriction2Via from '!!raw-loader!@site/docs/_examples/transportation/docusaurus/turn-restriction-02-via.yaml';
+import ExampleTurnRestriction2Connector1 from '!!raw-loader!@site/docs/_examples/transportation/docusaurus/turn-restriction-02-connector1.yaml';
+import ExampleTurnRestriction2Connector2 from '!!raw-loader!@site/docs/_examples/transportation/docusaurus/turn-restriction-02-connector2.yaml';
+
+
+# Roads
+
+In the Overture Transportation theme, a road is any kind of road,
+street, or path, including a dedicated path for walking or cycling
+(but excluding a railway). Road segments comprise the majority of
+ground-based transportation segments. Roads are modeled using
+[Segment](/reference/transportation/segment) features with the `subType`
+property value set to the value `road`.
+
+## Granularity and geometry
+
+Road segments support modeling the road network at a range of
+granularities. For example, a single road segment can represent:
+
+- A bidirectional street including all of its lanes and sidewalks.
+- A sidewalk by itself.
+- A one-way street or one direction of a dual carriageway.
+- A single lane or single section of a multi-lane highway.
+- A dedicated cycleway.
+- A hiking path.
+
+# Centerline and width
+
+A road segment's geometry represents the centerline of the section of
+road being modeled by the segment, with exactly half of the road's width
+occurring on each side of the segment at any given point along the
+segment geometry.
```

Comment:
I think a question this raised is what would count as the edge from the width perspective? 

I would suggest considering some guideline around the edge being the physical extent for associated travel mode. If its on a lane, it describes the physical extend of the lane, if its on a segment as a whole, it describes the width of the allowed travel modes such as car lanes + bike lanes + sidewalks.

Reply:
This is good feedback. I will delete the centerline section entirely and replace it with a sentence based on Ola's "virtual representation" wording that will go up in the section above.